tests/posix/fs/CMakeLists.txt

```diff
@@ -4,5 +4,11 @@ cmake_minimum_required(VERSION 3.13.1)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(fs)
 
-FILE(GLOB app_sources src/*.c)
-target_sources(app PRIVATE ${app_sources})
+target_sources(app PRIVATE src/main.c src/test_fat_mount.c)
+if (TEST_ONLY_OPEN_FLAGS)
```

Comment:
Why the need for this ifdeffery? I'd say the new testcase source file should be just added to the existing test application, without any conditions.

Reply:
OK, will separate testcases.

Reply:
Merged instead of separating but "ifdeffery" has been removed.

Reply:
Done, removed.